zen-oft-v2/scripts/demo-transfer-1.ts

```diff
@@ -0,0 +1,62 @@
+// This is a script for deploying your contracts. You can adapt it to deploy
+// yours, or create new ones.
+import hre from "hardhat";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const OPTIONS = "0x0003010011010000000000000000000000000000ea60";
```

Comment:
just curious what are the options we are passing?

Reply:
This is indicating that we are setting a gas limit of 6000 for the receive operation on the other side (`ea60` in the final is the number 6000). The other number set the type of the option. This is from the OFT Quickstart doc
![image](https://github.com/user-attachments/assets/d0f303dc-9e83-409e-a213-1cf72fe9929f)